js/services/AuthAPI.js

```diff
@@ -517,7 +517,9 @@ define(function(require, exports) {
 
     const executeWithRefresh = async function(httpPromise) {
         const result = await httpPromise;
-        await refreshToken();
+        if (config.userAuthenticationEnabled) {
+            await refreshToken();
+        }
         return result;
```

Comment:
This change is also unrelated to the referenced issue. This addition fixes a bug that prevents running Atlas when security is disabled.